### PR TITLE
Lift docs from Markdown to dartdoc

### PR DESCRIPTION
**Pull request by abarth**
I've also removed the top-level description of the Sky package. Instead, we
should host that content on flutter.io.

### CODE REVIEW

File: sky/packages/sky/lib/animation.dart [hunk 1]
```diff
@@ -2,9 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-/// The Sky animation engine.
+/// The Flutter animation engine.
 ///
-/// This library includes and re-exports all Sky animation classes.
+/// This library depend only on core Dart libraries and the `newton` package.
```
Comment (Hixie) [inline sky/packages/sky/lib/animation.dart]: depends

File: sky/packages/sky/lib/services.dart [hunk 2]
```diff
@@ -2,7 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-/// Services provided by Mojo
+/// Operating systems expose to Flutter apps
```
Comment (Hixie) [inline sky/packages/sky/lib/services.dart]: not sure what this means :-)

File: sky/packages/sky/lib/README.md [hunk 3]
```diff
@@ -1,71 +0,0 @@
-SKY SDK
```
Comment (Hixie) [inline sky/packages/sky/lib/README.md]: is this on flutter.io? we shouldn't lose it.

**Comment by Hixie** [comment]
LGTM

File: sky/packages/sky/lib/services.dart (hunk 2 above)
Comment (abarth) [inline sky/packages/sky/lib/services.dart]: Fixed.

File: sky/packages/sky/lib/README.md (hunk 3 above)
Comment (abarth) [inline sky/packages/sky/lib/README.md]: I have it locally, but I haven't pushed it yet.  I was going to replace the ascii art with a link to the flutter architecture slides before pushing it.

**Pull request merged**